### PR TITLE
Unbundle llvm

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+echo "export C_INCLUDE_PATH=$PREFIX/include"     >> compiler-wrapper
+echo "export CPLUS_INCLUDE_PATH=$PREFIX/include" >> compiler-wrapper
+chmod +x "compiler-wrapper"
+
+cp compiler-wrapper $CC
+cp compiler-wrapper $CXX
+echo "exec $BUILD_PREFIX/bin/$CC -L$PREFIX/lib \"\$@\""   >> $CC
+echo "exec $BUILD_PREFIX/bin/$CXX -L$PREFIX/lib \"\$@\""  >> $CXX
+
+export TF_SYSTEM_LIBS="llvm"
```
Comment (jjhelmus) [inline recipe/build.sh]: These additions may need to be moved inside the `if [[ "$target_platform" == linux* ]]; then` block as they do not look to work on macOS.

**Comment by jjhelmus** [comment]
Awesome work @isuruf.  Look like this speeds up the build considerably.  Why are `LIBDIR` and `INCLUDEDIR` set?

**Comment by jjhelmus** [comment]
> Why are `LIBDIR` and `INCLUDEDIR` set?

I'm guessing that this is because the [defaults](https://github.com/tensorflow/tensorflow/blob/master/.bazelrc#L130-L133) point at `/usr`.  This would explain the troubles I was having setting TF_SYSTEM_LIBS in the past where bazel would complain about including files outside its control.

**Comment by isuruf** [comment]
> I'm guessing that this is because the defaults point at /usr. This would explain the troubles I was having setting TF_SYSTEM_LIBS in the past where bazel would complain about including files outside its control.

That's a different issue. INCLUDEDIR is used when some headers used by tensorflow have different folder structure in system.

File: recipe/build.sh [hunk 2]
```diff
@@ -2,22 +2,43 @@
...
+export LIBDIR=$PREFIX/lib
+export INCLUDEDIR=$PREFIX/include
+
+echo "#!/bin/bash"                                > compiler-wrapper
+echo "export C_INCLUDE_PATH=$PREFIX/include"     >> compiler-wrapper
+echo "export CPLUS_INCLUDE_PATH=$PREFIX/include" >> compiler-wrapper
+chmod +x "compiler-wrapper"
+cp compiler-wrapper $CC
+cp compiler-wrapper $CXX
+echo "exec $BUILD_PREFIX/bin/$CC -L$PREFIX/lib \"\$@\""   >> $CC
```
Comment (isuruf) [inline recipe/build.sh]: @jjhelmus, this is how `-L$PREFIX/lib` is sent. This is a hack though. I tried passing `LIBRARY_PATH`, but that works only for gcc configured for native compilation.

**Comment by jjhelmus** [comment]
@isuruf Do you want to continue working on the osx build or is merging #80 with the reverted osx build alright?

**Comment by isuruf** [comment]
No, go ahead. I don't think I'll be able to fix the OSX build just using CI.

**Comment by jjhelmus** [comment]
> No, go ahead. I don't think I'll be able to fix the OSX build just using CI.

Sounds good, the OSX build can always be revisited.

**Pull request merged**